test/powershell/Language/Parser/Parsing.Tests.ps1

```diff
@@ -270,6 +270,7 @@ Describe 'splatting parsing' -Tags "CI" {
 }
 
 Describe 'Pipes parsing' -Tags "CI" {
+    ShouldBeParseError '|gps' EmptyPipeElement 0
```

Comment:
Do we need
```powershell
ShouldBeParseError '|' EmptyPipeElement 0
```

Reply:
The other two tests validate that something is before/after the pipe, and in a better way (it is clear what is missing/non-functional if one of those tests fail). With the proposed test, it would not be clear what became broken should it fail, so I feel it would add more harm than good.